htdocs/expedition/class/expedition.class.php

```diff
@@ -569,6 +572,56 @@ function fetch($id, $ref='', $ref_ext='', $ref_int='')
 		}
 	}
 
+	/**
+	 *	Classify the shipping as shipped
+	 *
+	 *	@return     int     <0 if ko, >0 if ok
+	 */
+	function ship()
+	{
+		global $conf,$langs,$user;
+
+		$error=0;
+
+		$this->db->begin();
+
+		$sql = 'UPDATE '.MAIN_DB_PREFIX.'expedition SET fk_statut=3';
+		$sql .= ' WHERE rowid = '.$this->id.' AND fk_statut = 2';
+
+		$resql=$this->db->query($sql);
+		if ($resql)
+		{
+			$this->statut=3;
+			
+			if (! $error)
+			{
+    				// Call trigger
+    				$result=$this->call_trigger('SHIPPING_SHIPPED',$user);
```

Comment:
SHIPPING_SHIPPED, perhaps it's better to use SHIPPING_MODIFY wich is standard trigger